include/Editor.h

```diff
@@ -57,6 +57,13 @@ class Editor : public QMainWindow
 	DropToolBar * addDropToolBar(QWidget * parent, Qt::ToolBarArea whereToAdd, QString const & windowTitle);
 
 	void closeEvent(QCloseEvent * event) override;
+	void keyPressEvent(QKeyEvent *ke) override;
+public slots:
+	/// Called by pressing the space key. Plays or stops.
```

Comment:
`//!` for documentation comments.

```suggestion
	//! Called by pressing the space key. Plays or stops.
```